accepted/2020/net5/net5.md

```diff
@@ -0,0 +1,502 @@
+with the duplication. But bottom line is that we need to make that change in
+MSBuild too.*
+
+* [@rainersigwald](https://github.com/rainersigwald): We've made it pretty far
+  with duplicated logic, but it results in really ugly MSBuild, since it's not a
+  very expressive programming language. It also creates the potential for drift
+  between the two definitions. That said, exposing it directly would create a
+  new tight coupling between MSBuild and NuGet that hasn't historically existed.
+  It would probably require a direct dependency and update flow plus coherency
+  requirements on both .NET Core SDK and VS insertions. If the logic were in a
+  separate package (we've talked about pushing it down to the framework at
+  various times) it'd be great to just expose that. With the logic in NuGet,
+  it's reasonable either way, just different tradeoffs. I'm amenable to exposing
+  a property function, but maybe we should go down the road of not doing it at
+  first.
+
+We're going to map the new entries as follows:
+
+| Framework      | Identifier    | Version | Profile |
+|----------------|---------------|---------|---------|
+| net48          | .NETFramework | 4.8     |         |
+| net5.0         | .NETCoreApp   | 5.0     |         |
+| net5.0-android | .NETCoreApp   | 5.0     | android |
+| net5.0-ios     | .NETCoreApp   | 5.0     | ios     |
+| net5.0-win     | .NETCoreApp   | 5.0     | win     |
+
+***Open Issue**. Please note that `net5.0`+ will map the TFI to `.NETCoreApp`.
+We need to announce this change so that package authors with custom .props and
+.targets are prepared. Link to DavKean’s doc on how to do it.*
+
+***Open Issue**. We should try to keep the TFI out of the .nuspec file. It seems
+NuGet uses the long form `.NETFramework,Version=4.5` in the dependency groups.
+We may want to change NuGet to allow the short form there as well and update our
+packaging tools to re-write to short name on pack.
+
+Specifically:
+
+* **We'll continue to use .NETCoreApp as the TFI**. This reduces the number of
+  pieces and build logic that needs to change in .NET SDK.
+
+* **net4x and earlier will continue to use .NETFramework as the TFI**. This
+  means that net4 and net5 aren't considered compatible by default, but the
+  compatibility will continue to be handled by the AssetTargetFallback in the
+  SDK/NuGet restore.
+
+* **We'll use the existing profile concept to encode the OS selection.** It's
+  worth noting that profiles today make things smaller whereas this use makes
+  things larger. We believe this can be handled in the NuGet layer by using this
+  rule for net5 and up. While that's not ideal, it reduces the number of
+  concepts we need to add.
+
+### TFMs are a closed set
+
+One question is whether third parties can extend the TFM space (i.e. the part
+after the dash) without having to rely on changes in NuGet/MSBuild. Due to the
+expansion of short name into TFI & TFM this would be non-trivial.
```

Comment:
Similar to how it works today, `TargetFramework` is a user-written friendly name and isn't used other than expanding out to the canonical version of TFI, TFV, TFP. Please do not change this.

Reply:
I'm not sure what you're asking? We're basically just saying that we don't want to support `net5.0-xxx` unless we know what `xxx` is.